app/main/posts/detail/post-detail.controller.js

```diff
@@ -71,6 +71,8 @@ function (
             $scope.$emit('setPageTitle', title);
         });
     }
+    angular.element(document.getElementById('bootstrap-app')).removeClass('hidden');
```

Comment:
besides the obvious error of only removing when we had a form id, I was waiting until fully loaded which isn't all that good , we need to be able to give the user feedback after we take them there. Also thinking this would be better handled as a general solution for when any route changes but I wasn't sure so I didn't want to go into that NOW .